### PR TITLE
Disables antialiasing for pixel icons

### PR DESCRIPTION
**Pull request by CyndaquilDAC**
pretty simple. forgive the old commits that were left over on my master branch

### CODE REVIEW

**Comment by Two-Ai** [comment]
FYI you can drop commits on a development branch by using [interactive rebase](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d).
(DO NOT do this on `master` or branches used by other people, only do this on development branches.)
```shell
# Add ninjamuffin99 as a remote
git remote add ninjamuffin99 https://github.com/ninjamuffin99/Funkin.git
# Checkout to the branch you want to rebase
git checkout no-more-pixel-antialiasing
# Rebase your changes onto ninjamuffin99's version of master
# This will open a text editor where you can specify the changes you want to make
# In this case, "drop" the commits you don't want
# (WARNING: this will overwrite your local branch, not reversible)
git rebase -i ninjamuffin99/master
# Replace the version of no-more-pixel-antialiasing you have on GitHub with the rebased local version
# (WARNING: this will overwrite the version of the branch on GitHub, not reversible)
git push --force-with-lease 
```
`git push --force[-with-lease]` is dangerous if you don't know what you're doing, so it's best to do the rebase before pushing the branch to github.

**Comment by CyndaquilDAC** [comment]
> FYI you can drop commits on a development branch by using [interactive rebase](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d).
> (DO NOT do this on `master` or branches used by other people, only do this on development branches.)
> 
> ```shell
> # Add ninjamuffin99 as a remote
> git remote add ninjamuffin99 https://github.com/ninjamuffin99/Funkin.git
> # Checkout to the branch you want to rebase
> git checkout no-more-pixel-antialiasing
> # Rebase your changes onto ninjamuffin99's version of master
> # This will open a text editor where you can specify the changes you want to make
> # In this case, "drop" the commits you don't want
> # (WARNING: this will overwrite your local branch, not reversible)
> git rebase -i ninjamuffin99/master
> # Replace the version of no-more-pixel-antialiasing you have on GitHub with the rebased local version
> # (WARNING: this will overwrite the version of the branch on GitHub, not reversible)
> git push --force-with-lease 
> ```
> 
> `git push --force[-with-lease]` is dangerous if you don't know what you're doing, so it's best to do the rebase before pushing the branch to github.

i am scared of this but thank you i will try this i pray i dont funk something up

**Comment by CyndaquilDAC** [comment]
i did that but it hasnt changed anything so idk what it did

**Comment by OPNA2608** [comment]
https://user-images.githubusercontent.com/23431373/114844937-951c7180-9ddb-11eb-8c0c-362081159274.mp4

…and `git push -f` afterwards.

File: source/HealthIcon.hx [hunk 1]
```diff
@@ -14,7 +14,6 @@ class HealthIcon extends FlxSprite
 		super();
 		loadGraphic(Paths.image('iconGrid'), true, 150, 150);
 
-		antialiasing = true;
```
Comment (ActualMandM) [inline source/HealthIcon.hx:17]: alternatively you could leave this here...

File: source/HealthIcon.hx [hunk 2]
```diff
@@ -35,6 +34,16 @@ class HealthIcon extends FlxSprite
 		animation.add('monster', [19, 20], 0, false, isPlayer);
 		animation.add('monster-christmas', [19, 20], 0, false, isPlayer);
 		animation.play(char);
+		switch(char){
```
Comment (ActualMandM) [inline source/HealthIcon.hx:37]: ...and do something like this instead
```haxe
		switch (char)
		{
			case 'bf-pixel' | 'senpai' | 'senpai-angry' | 'spirit':
				antialiasing = false;
		}
```

**Comment by CyndaquilDAC** [comment]
I deleted this fork why is this pr still active

**Pull request closed without merging**